extensions/pins-tutorial/manifest.json

```
@@ -0,0 +1,1279 @@
+{
+  "version": 1,
+  "locale": "en_US",
+  "platform": "4.4.3",
+  "metadata": {
+    "appmode": "rmd-static",
+    "primary_rmd": "pins-tutorial.Rmd",
+    "primary_html": null,
+    "content_category": null,
+    "has_parameters": false
+  },
+  "extension": {
+    "name": "pins-tutorial",
+    "title": "Pins Tutorial",
+    "description": "This R Markdown document provides a follow-along example showing how to create, retrieve, and use a pin.",
+    "homepage": "https://github.com/posit-dev/connect-extensions/tree/main/extensions/pins-tutorial",
+    "minimumConnectVersion": "2025.04.0",
+    "version": "0.0.1"
+  },
+  "environment": {
+    "r": {
+      "requires": ">=4.4"
+    }
```

Looks like the highest requirement of the packages is `R >=4.0`. Also suggesting the `~=` to stay on R version 4

```suggestion
    "r": {
      "requires": "~=4.0"
    }
```